async-voter.js

```diff
@@ -72,25 +73,45 @@ module.exports = (app, repository) => {
     const actions = payload.actions
     const text = payload.original_message.text
     const user = payload.user.name
+    const user_id = payload.user.id
     const channel_id = payload.channel.id
 
-    repository.get(channel_id, (err, reply) => {
+    const extractTicketDescription = (text) => {
+      return text.replace(/^<!here> ASYNC VOTE on \"/,'').replace(/"$/,'')
+    }
+
+    const ticket_description = extractTicketDescription(text)
+
+    repository.get(channel_id + "-" + ticket_description, (err, reply) => {
```

Comment:
does this `get` have a function any more since we are now storing each vote from each user separately - does anything get stored against this key that we actually need to retrieve?